.dependency_decisions.yml

```diff
@@ -0,0 +1,61 @@
+---
```

Comment:
I assume this was generated by the license finder, but man, this is kind of hard to parse with human eyes..